lib/nfc/protocols/mifare_classic.h

```diff
@@ -18,6 +19,7 @@
 #define MF_CLASSIC_ACCESS_BYTES_SIZE (4)
 
 typedef enum {
+    MfClassicTypeMini,
```

Comment:
This PR does not include a major API bump and does not require rebuilding .faps -- but public API methods accepting this enum will break. Since the binaries were built with an older version of this enum, with same members having different values. So, please either add the new entry to the end of the enum, or force the API version change.

Reply:
Oh, you are right. Fixing it! :+1: